src/transaction/log_manager.c

```diff
@@ -8390,6 +8399,8 @@ log_do_postpone (THREAD_ENTRY * thread_p, LOG_TDES * tdes, LOG_LSA * start_postp
 		    case LOG_DUMMY_CRASH_RECOVERY:
 		    case LOG_SMALLER_LOGREC_TYPE:
 		    case LOG_LARGER_LOGREC_TYPE:
+		    case LOG_START_ATOMIC_REPL:
```

Comment:
Same case here, don't log an error when finding start/end atomic replication. No action is required, so they belong to the same group as `LOG_SUPPLEMENTAL_INFO`.